zilencer/lib/stripe.py

```diff
@@ -92,6 +92,14 @@ def get_upcoming_invoice(stripe_customer_id: int) -> stripe.Invoice:
         print(''.join(['"upcoming_invoice": ', str(stripe_invoice), ',']))  # nocoverage
     return stripe_invoice
 
+def extract_current_subscription(stripe_customer: stripe.Customer) -> stripe.Subscription:
+    if not stripe_customer.subscriptions:
+        return None
+    for stripe_subscription in stripe_customer.subscriptions.data:
```

Comment:
We need to update our call to get_stripe_customer to retrieve the Subscription data, right? Using https://stripe.com/docs/api#expanding_objects?

Reply:
Nope. The customer object contains full subscriptions data. The attributes that requires expansion are marked separate in stripe API. 
https://stripe.com/docs/api#customer_object-subscriptions